src/test/java/ru/training/at/hw2/ex1/ExerciseFirst.java

```diff
@@ -0,0 +1,177 @@
+        softAssert.assertEquals(actualUserName, expectedUserName);
+
+        //5.Assert that there are 4 items on the header
+        //section are displayed and they have proper texts
+        WebElement homeButton = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By
+                        .cssSelector("a[href='index.html']"))));
+        String actualNameHomeButton = homeButton.getText();
+        String expectedNameHomeButton = "HOME";
+        softAssert.assertEquals(actualNameHomeButton, expectedNameHomeButton);
+        WebElement contactForm = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By
+                        .cssSelector("a[href='contacts.html']"))));
+        String actualContactForm = contactForm.getText();
+        String expectedContactForm = "CONTACT FORM";
+        softAssert.assertEquals(actualContactForm, expectedContactForm);
+        WebElement service = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By
+                        .cssSelector("a[class='dropdown-toggle']"))));
+        String actualServiceName = service.getText();
+        String expectedServiceName = "SERVICE";
+        softAssert.assertEquals(actualServiceName, expectedServiceName);
+        WebElement metalsAndColors = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By
+                        .xpath("//a[@href='metals-colors.html']"))));
+        String actualMetalAndColors = metalsAndColors.getText();
+        String expectedMetalAndColors = "METALS & COLORS";
+        softAssert.assertEquals(actualMetalAndColors, expectedMetalAndColors);
+
+        //6.Assert that there are 4 images on the Index Page and they are displayed
+        WebElement imageFirst = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By
+                        .cssSelector("span[class='icons-benefit icon-practise']"))));
+        softAssert.assertTrue(imageFirst.isDisplayed());
+        WebElement imageSecond = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By
+                        .cssSelector("span[class='icons-benefit icon-custom']"))));
+        softAssert.assertTrue(imageSecond.isDisplayed());
+        WebElement imageThird = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By
+                        .cssSelector("span[class='icons-benefit icon-multi']"))));
+        softAssert.assertTrue(imageThird.isDisplayed());
+        WebElement imageFourth = (new WebDriverWait(webDriver, 10)
+                .until(ExpectedConditions.presenceOfElementLocated(By
+                        .cssSelector("span[class='icons-benefit icon-base']"))));
+        softAssert.assertTrue(imageFourth.isDisplayed());
```

Comment:
I suppose here elements could be collected to the `List<WebElement>` and you could use loop for the verification visibility of them